src/documents/barcodes.py

```diff
@@ -279,6 +292,54 @@ def asn(self) -> Optional[int]:
 
         return asn
 
+    @property
+    def tags(self) -> Optional[list[int]]:
+        """
+        Search the parsed barcodes for any tags.
+        Returns the detected tag ids (or empty list)
+        """
+        tags = []
+
+        # Ensure the barcodes have been read
+        self.detect()
+
+        for x in self.barcodes:
+            tag_texts = x.value
+
+            for raw in tag_texts.split(","):
+                try:
+                    tag = None
+
+                    mappings = settings.CONSUMER_TAG_BARCODE_MAPPING.items()
```

Comment:
Are you asking
1. why I put it into the 'mappings' variable? I guess I wanted to avoid a long line. I can change it. Or
2. Why I use .items() to get key and value together als loop variables? I do this to avoid the value-by-key lookup. Happy to change as well if you prefer. Let me know.

Reply:
To me it would be more clear using option 2

Reply:
@pkrahmer i believe this is the only outstanding issue here, if im reading the comments correctly

Reply:
@shamoon fixed.